site/content/blog/2020-04-17-zap-jwt-scanner/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "ZAP JWT Scanner"
+description: "How to scan JWT endpoints using OWASP ZAP"
+images:
+- https://www.zaproxy.org/blog/2020-04-17-zap-jwt-scanner/images/jwt-options-panel.png
+type: post
+tags:
+- blog
+date: "2020-04-17"
+addSocialPreview: true
+authors:
+    - preetkaran20 (Karan Preet Singh Sasan)
+---
+
+With the popularity of JSON Web Tokens (JWTs)there comes the need to secure their use so that they are not misused because of bad configuration, older libraries, or buggy implementations. So the JWT Scanner Addon is used to find such vulnerabilities and this blog explains on how to use it.
```

Comment:
Sorry it looks like I introduced an error here: 
```suggestion
With the popularity of JSON Web Tokens (JWTs) there comes the need to secure their use so that they are not misused because of bad configuration, older libraries, or buggy implementations. So the JWT Scanner Add-on is used to find such vulnerabilities and this blog explains on how to use it.
```